webapi/src/main/scala/org/knora/webapi/http/status/ApiStatusCodesV2.scala

```diff
@@ -22,21 +22,25 @@ object ApiStatusCodesV2 {
    */
   def fromException(ex: Throwable): StatusCode =
     ex match {
-      // Subclasses of RequestRejectedException (which must be last in this group)
+      // Subclasses of RequestRejectedException
       case NotFoundException(_)           => StatusCodes.NotFound
       case ForbiddenException(_)          => StatusCodes.Forbidden
       case BadCredentialsException(_)     => StatusCodes.Unauthorized
       case DuplicateValueException(_)     => StatusCodes.BadRequest
       case OntologyConstraintException(_) => StatusCodes.BadRequest
       case EditConflictException(_)       => StatusCodes.Conflict
-      case RequestRejectedException(_)    => StatusCodes.BadRequest
       case BadRequestException(_)         => StatusCodes.BadRequest
+      case ValidationException(_, _)      => StatusCodes.BadRequest
+      case RequestRejectedException(_)    => StatusCodes.BadRequest
+      // RequestRejectedException must be last in this group
 
-      // Subclasses of InternalServerException (which must be last in this group)
+      // Subclasses of InternalServerException
       case UpdateNotPerformedException(_)    => StatusCodes.Conflict
       case TriplestoreTimeoutException(_, _) => StatusCodes.GatewayTimeout
       case InternalServerException(_)        => StatusCodes.InternalServerError
-      case _                                 => StatusCodes.InternalServerError
+      // InternalServerException must be last in this group
```

Comment:
Same question. Out of curiosity.

```suggestion
      // InternalServerException must be the last one in this group
```